src/index.js

```diff
@@ -1,9 +1,19 @@
 import loadScripts from './analytics';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {  BrowserRouter, Route, Switch } from 'react-router-dom';
+import { ROOT, TEAM_PAGE } from './routes/index';
 import App from './containers/App';
+import Team from './containers/Team';
 import registerServiceWorker from './registerServiceWorker';
 
 loadScripts();
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render((
```

Comment:
👍

Reply:
The footer should be updated with the link to the team page, no ?

Reply:
Done!